docs/runbooks/KubeVirtDeprecatedAPIsRequested.md

```diff
@@ -0,0 +1,36 @@
+# KubeVirtDeprecatedAPIsRequested
+<!-- Edited by fmatouschek, May 2023-->
+
+## Meaning
+
+This alert fires when a deprecated KubeVirt API was requested.
+
+## Impact
+
+Usage of deprecated APIs is not recommended because they will be removed in a future release.
```

Comment:
@0xFelix I would also add what will happen if they use a deprecated API after it was already removed.

Reply:
will it just not do what it should? Will it create an error?

Reply:
Once removed any attempt to create a new object or call a subresource API using this version will fail with an error yes.

Reply:
@sradco would you still like more context in the impact here or is this okay now?

Reply:
Will leave this for the author decision.

Reply:
Would this work?

"Using a deprecated API is not recommended because the request will fail when the API is removed in a future release."